### PR TITLE
Add role and binding if OOB runtime inventory config is enabled

### PR DESCRIPTION
**Pull request by dakaneye**
Signed-off-by: Samuel Dacanay <sam.dacanay@anchore.com>

### CODE REVIEW

**Review by robertp — APPROVED** [review]
This LGTM but I also want @Btodhunter's eyes on it.

File: stable/anchore-engine/templates/catalog_deployment.yaml [hunk 1]
```diff
@@ -254,3 +254,29 @@ spec:
...
+- apiGroups: [""]
+  resources: ["pods","namespaces"]
+  verbs: ["get", "watch", "list"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: catalog-cluster-read-only-binding
+subjects:
```
Comment (Btodhunter) [inline stable/anchore-engine/templates/catalog_deployment.yaml:274]: @dakaneye I believe that this is going to need a service account to bind to, which the chart doesn't currently create.

**Review by Btodhunter — CHANGES_REQUESTED** [review]

File: stable/anchore-engine/templates/catalog_deployment.yaml (hunk 1 above)
Comment (Btodhunter) [reply stable/anchore-engine/templates/catalog_deployment.yaml:274]: The chart does let you specify a service account to run deployments as, but nothing in the chart actually creates the service account. I think that if we have a thing that _requires_ a service account, we should create it in the chart.

Comment (dakaneye) [reply stable/anchore-engine/templates/catalog_deployment.yaml:293]: I added one for the catalog specifically!

**Review by Btodhunter — CHANGES_REQUESTED** [review]

File: stable/anchore-engine/values.yaml [hunk 2]
```diff
@@ -535,6 +535,13 @@ anchoreCatalog:
     annotations: {}
     labels: {}
 
+    # These are the service account parameters for the catalog deployment
```
[the remark this answers is not in the document]
Comment (dakaneye) [reply stable/anchore-engine/values.yaml:538]: Fixed!

**Pull request closed without merging**